web/views/exposure.py

```diff
@@ -38,6 +38,8 @@ def form_valid(self, form):
         """If the form is valid, save the associated model and add to the exposure"""
         self.object = form.save(commit=False)
         self.object.dataset = self.dataset
+        rems_forms_dict = get_rems_forms(self.object.form_id)
```

Comment:
Should be `rems_forms_list` to be consistent

Reply:
this is actually a dict because i call it with the form id, so the api returns a dict with details of the form with that id